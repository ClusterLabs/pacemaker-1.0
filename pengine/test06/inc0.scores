Allocation scores:
clone_color: rsc1 allocation score on node1: 0
clone_color: rsc1 allocation score on node2: 0
clone_color: child_rsc1:0 allocation score on node1: 0
clone_color: child_rsc1:0 allocation score on node2: 0
clone_color: child_rsc1:1 allocation score on node1: 0
clone_color: child_rsc1:1 allocation score on node2: 0
clone_color: child_rsc1:2 allocation score on node1: 0
clone_color: child_rsc1:2 allocation score on node2: 0
clone_color: child_rsc1:3 allocation score on node1: 0
clone_color: child_rsc1:3 allocation score on node2: 0
clone_color: child_rsc1:4 allocation score on node1: 0
clone_color: child_rsc1:4 allocation score on node2: 0
native_color: child_rsc1:0 allocation score on node1: 0
native_color: child_rsc1:0 allocation score on node2: 0
native_color: child_rsc1:1 allocation score on node1: 0
native_color: child_rsc1:1 allocation score on node2: 0
native_color: child_rsc1:2 allocation score on node1: 0
native_color: child_rsc1:2 allocation score on node2: 0
native_color: child_rsc1:3 allocation score on node1: -1000000
native_color: child_rsc1:3 allocation score on node2: 0
native_color: child_rsc1:4 allocation score on node1: -1000000
native_color: child_rsc1:4 allocation score on node2: -1000000
