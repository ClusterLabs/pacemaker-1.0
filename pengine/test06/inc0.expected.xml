<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="1" num_updates="1" dc-uuid="0" have-quorum="false" remote-tls-port="0" validate-with="pacemaker-1.0">
  <configuration>
    <crm_config><cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair" name="no-quorum-policy" value="ignore"/></cluster_property_set></crm_config>
    <nodes>
      <node id="uuid1" uname="node1" type="member"/>
      <node id="uuid2" uname="node2" type="member"/>
    </nodes>
    <resources>
      <clone id="rsc1"><meta_attributes id="clone-rsc1.meta"><nvpair id="nvpair" name="resource-stickiness" value="1"/><nvpair id="nvpair" name="clone-max" value="5"/><nvpair id="nvpair" name="clone-node-max" value="2"/></meta_attributes><primitive id="child_rsc1" class="heartbeat" type="apache"><meta_attributes id="primitive-child_rsc1.meta"/></primitive></clone>
    </resources>
    <constraints>
    </constraints>
  </configuration>
  <status>
    <node_state id="uuid1" ha="active" uname="node1" crmd="online" join="member" expected="member" in_ccm="true">
      <transient_attributes id="transient_attributes">
        <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/></instance_attributes>
      </transient_attributes>
    </node_state>
    <node_state id="uuid2" ha="active" uname="node2" crmd="online" join="member" expected="member" in_ccm="true">
      <transient_attributes id="transient_attributes">
        <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/></instance_attributes>
      </transient_attributes>
    </node_state>
  </status>
</cib>
